soc/ite/ec/it8xxx2/Kconfig.soc

```diff
@@ -36,45 +23,44 @@ config SOC_IT8XXX2_REG_SET_V2
 	  This option is selected by a variable of which soc, and will
 	  determine the register for the IT82xx2 specification.
 
-if SOC_IT8XXX2
-
-choice IT8XXX2_SERIES
-	prompt "IT8XXX2 Series"
-	default SOC_IT81302_BX
-
 config SOC_IT81302_BX
-	bool "IT81302 BX version"
+	bool
+	select SOC_IT8XXX2
 	select SOC_IT8XXX2_REG_SET_V1
```

Comment:
`select SOC_IT8XXX2_REG_SET_V1` to `Kconfig`

Reply:
done, moved all this stuff